docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,150 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`your first plugin <first-plugin>`, maybe you
+want to write a more interesting command than the basic ``.hello`` one. Not
+that there is anything wrong with that command! The emoticons plugin is
+composed of commands like this one::
+
+    @plugin.command('shrug')
+    @plugin.action_command('shrugs')
+    def shrug(bot, trigger):
+        bot.say('¯\\_(ツ)_/¯')
+
+Which is one of the maintainers' favorite commands to use. However, let's see
+if we can do something a bit more *complex* than that.
+
+
+Greeting a user by their name
+=============================
+
+Have you noticed that a :ref:`plugin callable <Plugin callables>` takes **two
+arguments?** The first one is the ``bot``, i.e. an instance of Sopel (wrapped
+for your convenience) that you can use to
+:doc:`interact with the bot </plugin/bot>`.
+
+In the previous tutorial, we used ``bot.reply``, which is convenient but not
+always what you want. Maybe you want the bot to say this::
```

Comment:
Nit: elaborate on what `convenient` is getting at here, the reader may not be familiar enough with what it does

```suggestion
In the previous tutorial, we used ``bot.reply``, which is convenient when             
responding directly to a user, but not always what you want. Maybe you want the       
bot to say something more complex::
```

Reply:
I'd say "can be convenient", even, instead of the blanket "is".